src/Symfony/Component/Message/Transport/Serialization/SymfonySerialization.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message\Transport\Serialization;
+
+use Symfony\Component\Serializer\SerializerInterface;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class SymfonySerialization implements DecoderInterface, EncoderInterface
```

Comment:
I would call it just `Serializer` (as we do for `Validator` etc in other parts of the framework IIRC).

Reply:
final?

Reply:
That's a good point. Done in ae3800828a.